docs/lib/restapi/fragments/flutter/fetch.md

```diff
@@ -0,0 +1,76 @@
+## GET requests
+
+To make a GET request, first build a RestOptions object and then use the Amplify.API.get api to issue the request:
+
+```dart
+void getTodo() {
```

Comment:
Let's not wrap the code snippet in a method if we are not using it properly. (a `getTodo` returning a void is not a good code example)

Reply:
sounds good, removed the method declaration, it's now just a try/catch code example